src/components/Modal/index.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { Row } from '../Containers';
+
+const radius = '10px';
+
+export const ModalHeader = styled(Row)`
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem;
+  background: white;
+  width: 100%;
+  border-top-left-radius: ${radius};
+  border-top-right-radius: ${radius};
+`;
+
+export const ModalBody = styled.div`
+  display: flex;
+  padding: 0.5rem;
+  background: white;
```

Comment:
**Copied from doskachek:**
I think it would be better to move these color values to the theme.